tools/ci_build/build.py

```diff
@@ -227,7 +227,7 @@ def install_hosting_deps(source_dir):
         boostrap_path = os.path.join(vcpkg_folder_path, 'bootstrap-vcpkg' + file_ending)
         run_subprocess([boostrap_path])
 
-    run_subprocess([vcpkg_executable, 'install', 'boost-beast', 'boost-program-options', 'rapidjson'])
+    run_subprocess([vcpkg_executable, 'install', 'boost-beast', 'boost-program-options', 'boost-uuid'])
```

Comment:
If we could use libuuiddev, we may get rid of this new dependency.

Reply:
As discussed we may change this with feedback from onnxruntime team